public/js/filterbar.js

```diff
@@ -34,18 +33,18 @@ $(function() {
 				}
 				return true;
 			}
-		}).show();
-
+		});
+		filteredRows.show();
+		var emptySearch = $('.empty-search');
+		filteredRows.length === 0 ? emptySearch.attr('aria-hidden', 'false') : emptySearch.attr('aria-hidden', 'true');
 		return rows;
 	}
 
 	function navigateRows(key) {
-
-
-		var rows = $componentList.find(itemSelector + ':visible'),
-			current = rows.filter('.focused'),
-			index = rows.index(current),
-			select = 0;
+		var rows = $componentList.find(itemSelector + ':visible');
+		var	current = rows.filter('.focused');
```

Comment:
Odd spacing here, the previous code you've replaced was ok, was it changed based on linting? 

Would be good to either put it back to the format it was in, or sort out the whitespace